tests/LoginTrait.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Tests;
+
+use App\Infrastructure\Security\User;
+use Symfony\Component\BrowserKit\Cookie;
+use App\Infrastructure\Test\Adapter\Repository\ParticipantRepository;
+use Symfony\Component\Security\Core\Authentication\Token\UsernamePasswordToken;
+
+/**
+ * Trait LoginTrait
+ * @package App\Tests
+ */
+trait LoginTrait
+{
+    /**
+     * @return void
+     */
+    private function logIn()
+    {
+        $session = self::$container->get('session');
+
```

Comment:
Il faut dans un premier temps clear les cookies `$this->client->getCookieJar()->clear()`.